rosa_getting_started/rosa-quickstart.adoc

```diff
@@ -0,0 +1,156 @@
+:_content-type: ASSEMBLY
+[id="rosa-quickstart"]
+= {product-title} quickstart guide
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-quickstart
+
+toc::[]
+
+[NOTE]
+====
+If you are looking for a comprehensive getting started guide for ROSA, see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Comprehensive guide to getting started with {product-title}]
+====
+
+Follow this getting started document to quickly create a {product-title} (ROSA) cluster, grant user access, deploy your first application, and learn how to revoke user access and delete your cluster.
+
+The procedures in this document enable you to create a cluster that uses AWS STS. For more information about using AWS STS with ROSA clusters, see xref:../rosa_architecture/rosa-understanding.adoc#rosa-using-sts_rosa-understanding[Using the AWS Security Token Service].
+
+[id="rosa-getting-started-prerequisites_{context}"]
+== Prerequisites
```

Comment:
That's a good question, and I'm not sure. I'm inclined to leave them for now, but let's leave this open for more feedback.

Reply:
The "Using the AWS Security Token Service" link directs users to here: https://51818--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-understanding.html#rosa-using-sts_rosa-understanding

I think it'd be better if it directed users to the "Credential modes" section underneath the About Rosa section (ie, here: https://51818--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-understanding.html#rosa-understanding-credential-modes_rosa-understanding). I was a bit confused when I clicked on the link and instead of immediately seeing something about STS, I had to scroll down.

Reply:
Good catch! I'm updating that link in both the quickstart and getting started guides